test/lint/pass_lint_json.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -o errexit
```

Comment:
Can you leave a blank line after the hash bang, like other scripts? Same in the other ones you added here